projects/npm-tools/packages/npm-scripts/contrib/prettier/prettier.js

```diff
@@ -1106,31 +1145,30 @@ function prepare(filepath) {
 			file = fallback;
 		}
 	}
-	else {
-		root = getFrontendProjectsRoot(filepath);
 
-		if (root) {
-			const scripts = path.join(
-				root,
-				'projects/npm-tools/packages/npm-scripts'
-			);
+	const frontendProjectsRoot = getFrontendProjectsRoot(filepath);
+
+	if (frontendProjectsRoot) {
+		const scripts = path.join(
+			frontendProjectsRoot,
+			'projects/npm-tools/packages/npm-scripts'
+		);
 
-			const wrapper = path.join(scripts, 'src/scripts/prettier.js');
+		const wrapper = path.join(scripts, 'src/scripts/prettier.js');
 
-			if (fs.existsSync(wrapper)) {
-				dir = scripts;
-				file = wrapper;
-			}
+		if (fs.existsSync(wrapper)) {
+			dir = scripts;
+			file = wrapper;
 		}
-		else {
-			const extension = require('vscode').extensions.getExtension(
-				'esbenp.prettier-vscode'
-			);
-
-			if (extension) {
-				dir = path.join(extension.extensionPath);
-				file = path.join(dir, 'node_modules/prettier');
-			}
+	}
+	else {
+		const extension = require('vscode').extensions.getExtension(
+			'esbenp.prettier-vscode'
+		);
+
+		if (extension) {
+			dir = path.join(extension.extensionPath);
+			file = path.join(dir, 'node_modules/prettier');
 		}
 	}
```

Comment:
I think we are always ending up in this `else` branch and thus always taking the extensions version of prettier

Reply:
Yeah I think this is the case why you don't see it working properly, I just tested locally and I am always ending up in this else block and not using our custom prettier rules